src/core/ext/xds/xds_client_stats.h

```diff
@@ -63,7 +63,11 @@ class XdsLocalityName : public RefCounted<XdsLocalityName> {
   XdsLocalityName(std::string region, std::string zone, std::string sub_zone)
       : region_(std::move(region)),
         zone_(std::move(zone)),
-        sub_zone_(std::move(sub_zone)) {}
+        sub_zone_(std::move(sub_zone)),
+        locality_labels_(
+            std::make_shared<std::map<std::string, std::string>>()) {
```

Comment:
I'm guessing that this is being saved as an std::make_shared<std::map<std::string, std::string>> instead of just a `std::string` for the actual value to avoid the per-call map creation?

Reply:
Yes, exactly.